src/pages/cli/storage/override.mdx

```diff
@@ -14,46 +15,70 @@ The command creates a new `overrides.ts` file under `amplify/backend/storage/<re
-|[s3GuestPublicPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html)|The IAM policy for guest users' write access to `public/*` prefix|
-|[s3GuestUploadPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html)|The IAM policy for guest users' write access to `uploads/*` prefix|
-|[s3GuestReadPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html)|The IAM policy for guest users' read access|
+| Amplify-generated resource | Description |
+| --- | --- |
+| [s3Bucket](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html) | The S3 bucket that Amplify generates for file storage upon `amplify add storage` |
+| [s3AuthPublicPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for authenticated users' write access to `public/*` prefix |
+| [s3AuthProtectedPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for authenticated users' write access to `protected/*` prefix |
+| [s3AuthPrivatePolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for authenticated users' write access to `private/*` prefix |
+| [s3AuthUploadPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for authenticated users' write access to `uploads/*` prefix |
+| [s3AuthReadPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for authenticated users' read access |
+| [s3GuestPublicPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for guest users' write access to `public/*` prefix |
+| [s3GuestUploadPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for guest users' write access to `uploads/*` prefix |
+| [s3GuestReadPolicy](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html) | The IAM policy for guest users' read access |
+
+<br />
+
+For example, to use storage override to update the S3 storage public Auth policy for S3 PUT and GET access, use the following example:
+
+```ts
+import { AmplifyS3ResourceTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyS3ResourceTemplate) {
+  resources.s3AuthPublicPolicy.policyDocument.Statement =  [
+    ...resources.s3AuthPublicPolicy.policyDocument.Statement, 
+    {
+      Effect: "Allow",
+      Action: [ "s3:PutObject","s3:PutObjectAcl","s3:GetObject"], 
+      Resource: "<ARN of the S3 resource>"
```

Comment:
shouldn't I be able to reference the S3 resource through the `resources` variable?

Reply:
yes, the example here updates the existing policy by keep the existing statements and appending an inline policy.